src/Core/System/Resources/config/listing.xml

```diff
@@ -72,5 +72,13 @@
             <helpText lang="de-DE">Die Anzahl der Produkte die auf einer Suchseite oder in einer Produktliste je Seite in der Storefront angezeigt werden.</helpText>
             <min>1</min>
         </input-field>
+
+        <input-field type="text">
+            <name>limits</name>
+            <label>Limits in listing</label>
+            <label lang="de-DE">Limits in listing</label>
+            <helpText>Limiter for products per page in listing, divided by pipe, for example `6|12|18|24|48`</helpText>
```

Comment:
Don't we need a migration to set the default config?

Reply:
If we create a migration with a default value the feature is opt-out instead of opt-in, I don't know if that is desirable

Reply:
Ahh get it 👍🏽, so with no system config & slot config, the dropdown will be hidden? I'd hint it in the help text